packages/common/test/mocha-hooks.ts

```diff
@@ -0,0 +1,34 @@
+let testSuites = 0
```

Comment:
This file should be moved to a common folder (`/common/test/mocha-hooks.ts`), and not be copied on each subpackage.